api/reg.py

```diff
@@ -524,7 +525,11 @@ def beginFile(self, genOpts):
         #
         # User-supplied prefix text, if any (list of strings)
         if (genOpts.prefixText):
+            git_rev = subprocess.check_output(['git', 'rev-parse', '--short=10', 'HEAD']).decode('utf-8').strip()
```

Comment:
I'd prefer it generate an empty string, or "Unknown", or something of that nature, rather than failing.